google_images_download/google_images_download.py

```diff
@@ -99,7 +99,7 @@ def run(self, item, filename, filename_format='basename', no_clobber=True):
         """
         try:
             req = Request(item, headers={"User-Agent": self.ua.firefox})
-            with urlopen(req) as response, \
+            with urlopen(req, context=ssl._create_unverified_context()) as response, \
```

Comment:
Use try/except for the default method and catch ssl.CertificateError? That way the default is still used until error. 

Also add debug logging when catching ssl.CertificateError

Reply:
can I merge this?

Reply:
yes, just wait for @rytoj